stdpopsim/catalog/DroMel/dfes.py

```diff
@@ -41,20 +41,78 @@ def _HuberDFE():
     # p. 2 in supplement says that the total sequence length of synonymous sites LS
     # related to the total sequence length of nonsynonymous sites LNS by LNS = 2.85 * LS;
     # so this is 1 / (1 + 2.85) = 0.2597402597402597
-    prop_synonymous = 0.26
```

Comment:
I get an error running the PR locally. Looks like this got cut and pasted instead of copy and pasted?